src/mkindex_options.hpp

```diff
@@ -187,10 +187,12 @@ void parseCommandLine(LambdaIndexerOptions & options, int argc, char const ** ar
     parser.parse();
 
     // set db index type
-    if (dbIndexTypeTmp == "bifm")
+    if (dbIndexTypeTmp == "fm")
+        options.indexFileOptions.indexType = DbIndexType::FM_INDEX;
+    else if (dbIndexTypeTmp == "bifm")
         options.indexFileOptions.indexType = DbIndexType::BI_FM_INDEX;
     else
-        options.indexFileOptions.indexType = DbIndexType::FM_INDEX;
+        throw seqan3::argument_parser_error("ERROR: Unknown index type \"" + dbIndexTypeTmp + "\"");
```

Comment:
argument parser already checks the allowed values

Reply:
I just noticed, this is not true. `index_file_options::indexType` has no default initialized value, I put it there....no clue what I was thinking. :thinking: